### DISCUSSION TITLE
Is there a repo to report issues about the Marketplace?

### DISCUSSION THREAD

**Discussion by sketchbuch** [microsoft/vscode-discussions#1934] [Extension Development QnA] [answered]
I found a problem with the rendering of lists in markdown on the overview tab after you go to the changelog/ratings tab

**Comment by alefragnani** [comment, marked as answer]
Hi @sketchbuch ,

You may try their feedback repo https://github.com/microsoft/vsmarketplace, or via email vsmarketplace@microsoft.com.

Hope this helps
